src/sparseml/pytorch/torchvision/train.py

```diff
@@ -51,8 +51,9 @@ def train_one_epoch(
     optimizer.zero_grad()
 
     header = f"Epoch: [{epoch}]"
-    for i, (image, target) in enumerate(
-        metric_logger.log_every(data_loader, args.print_freq, header)
+    for i, (image, target) in zip(
+        range(args.max_train_steps),
```

Comment:
would prefer to have an explicit if/break clause for readability, especially since it's already using `enumerate`

Reply:
this also let's us avoid needing to overwrite a default for max train steps

Reply:
Check new logic